File: app/src/main/java/com/example/brochures/network/parsingobjects/Content.java
package com.example.brochures.network.parsingobjects;

import com.squareup.moshi.Json;

public class Content {

    @Json(name = "placement")
    private String placement;
    @Json(name = "adFormat")
    private String adFormat;
    @Json(name = "contentFormatSource")
    private String contentFormatSource;
    @Json(name = "contentType")
    private String contentType;
    @Json(name = "externalTracking")
    private ExternalTracking externalTracking;
    @Json(name = "content")
    private Content__1 content;

    public String getPlacement() {
        return placement;
    }

    public void setPlacement(String placement) {
        this.placement = placement;
    }

    public String getAdFormat() {
        return adFormat;
    }

    public void setAdFormat(String adFormat) {
        this.adFormat = adFormat;
    }

    public String getContentFormatSource() {
        return contentFormatSource;
    }

    public void setContentFormatSource(String contentFormatSource) {
        this.contentFormatSource = contentFormatSource;
    }

    public String getContentType() {
        return contentType;
    }

    public void setContentType(String contentType) {
        this.contentType = contentType;
    }

    public ExternalTracking getExternalTracking() {
        return externalTracking;
    }

    public void setExternalTracking(ExternalTracking externalTracking) {
        this.externalTracking = externalTracking;
    }

    public Content__1 getContent() {
        return content;
    }

    public void setContent(Content__1 content) {
        this.content = content;
    }

}

File: app/src/main/java/com/example/brochures/network/parsingobjects/Content.kt
package com.example.brochures.network.parsingobjects

import com.example.brochures.network.SingleToArray
import com.squareup.moshi.Json

class Content {
    @Json(name = "placement")
    var placement: String? = null

    @Json(name = "adFormat")
    var adFormat: String? = null

    @Json(name = "contentFormatSource")
    var contentFormatSource: String? = null

    @Json(name = "contentType")
    var contentType: String? = null

    @Json(name = "externalTracking")
    var externalTracking: ExternalTracking? = null

    @Json(name = "content")
    //todo @SingleToArray
    val innerContentWrapper: InnerContentWrapper? = null //List<InnerContent>? = null //todo  List<Content__1>? = null
}